doc/guide/hpt.hpp

```diff
@@ -1,7 +1,7 @@
-namespace mlpack {
+ens::namespace mlpack {
```

Comment:
I'm a bit confused by this... regex gone bad?

Reply:
I do not remember what convinced me to do this, but reverting it seems to break:
```
Generating docs for page hpt_guide...
/home/ryan/Documents/mlpack/doc/guide/hpt.hpp:35: error: unable to resolve reference to `ens::GridSearch' for \ref command (warning treated as error, aborting now)
Exiting...
CMakeFiles/doc.dir/build.make:57: recipe for target 'CMakeFiles/doc' failed
```

Reply:
Oh, you can probably remove any `\ref`s to anything in the `ens` namespace in `hpt.hpp` then.  If it's still complaining about something like `ens::GridSearch`, you could also just do `\c ens::GridSearch` and I think that would work too. :+1:

Reply:
That works!